README.md

```diff
@@ -580,6 +580,22 @@ cancellation of all outstanding activities.
 The `shutdown()` invocation will wait on all activities to complete, so if a long-running activity does not at least
 respect cancellation, the shutdown may never complete.
 
+### Workflow Replay
+
+Given a workflow's history, it can be replayed locally to check for things like non-determinism errors. For example,
+assuming `history_json_str` is populated with a JSON string history either exported from the web UI or from `tctl`, the
+following function will replay it:
+
+```python
+from temporalio.worker import Replayer
+
+async def run_replayer(history_json_str: str):
+  replayer = Replayer(workflows=[SayHello])
+  await replayer.replay_workflow(history_json_str)
```

Comment:
The reason I am not including that is because we don't yet have a high-level way so they'd have to paginate and build the event list themselves. So there actually is no easy way to get history from a client. I considered an overload/alternative that took a client that would load the history for you, but I think I'll just wait for the high-level history event iterator.